packages/url_launcher/url_launcher/test/url_launcher_test.dart

```diff
@@ -205,6 +205,21 @@ void main() {
       await launchResult;
       expect(binding.renderView.automaticSystemUiAdjustment, isTrue);
     });
+
+    test('sets automaticSystemUiAdjustment to not be null', () async {
+      final TestWidgetsFlutterBinding binding =
+          TestWidgetsFlutterBinding.ensureInitialized();
+      debugDefaultTargetPlatformOverride = TargetPlatform.android;
+      expect(binding.renderView.automaticSystemUiAdjustment, isNotNull);
+      final Future<bool> launchResult =
+          launch('http://flutter.dev/', statusBarBrightness: Brightness.dark);
+
+      // The automaticSystemUiAdjustment should be set before the launch
+      // and not null after the launch result is complete.
+      expect(binding.renderView.automaticSystemUiAdjustment, isNotNull);
```

Comment:
do you want to be more precise here and below and expect that this is `true`?

Reply:
Yes it should be true. I updated and ran the test again.